azavea/iow-boundary-tool
# Force users to reset password on first login

## Overview

Force users to reset their password on first login.

Closes #67 


### Notes
- Using the Django admin to reset a password does not set `has_admin_generated_password` to False, so that flow will not force the user to reset their password. If that is desired, that could be a follow-up card.

## Testing Instructions

- `scripts/resetdb`
- `scripts/server`
- Visit django admin at localhost:8181/admin and add a user
- Login as that user, ensure redirected to Reset password page
- [ ] Reset password and ensure login works
- [ ] Login again, ensure not asked to reset again
- [ ] Login as development users (e.g. v1@azavea.com), ensure that no resetting is required

## Checklist
